tools/e2etesting/MqttTestValidator/MqttValidation/MqttValidationTests.cs

```diff
@@ -0,0 +1,72 @@
+using FluentAssertions;
+using NUnit.Framework;
```

Comment:
typically the using statements are within the namespace block

Reply:
Done